ui_test/POM_hiway_asg/pages/loging_using_google.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from POM_hiway_asg.pages.constant_var import LOGIN_WITH_GOOGLE_BTN, USER_NAME_IDENTIFIER, PASSWORD_IDENTIFIER, \
+    START_USING_HIWAY_BTN, TIMESHEET_TAB_BTN, VALID_USERNAME_LOGIN, VALID_PASSWORD_LOGIN
+from selenium.webdriver.common.keys import Keys
+from selenium.common.exceptions import NoSuchElementException
+
+logging.basicConfig(filename='login_using_google_log.log',
+                    format='%(asctime)s: %(levelname)s %(message)s',
+                    datefmt='%d/%m/%y %I:%M:%S %p')
+
+
+class LoginWithGoogle():
+    logger = logging.getLogger()
+    logger.setLevel(logging.ERROR)
+
+    def __init__(self, driver):
+        self.driver = driver
+
+    def click_login_with_google_btn(self):
+        self.driver.find_element_by_xpath(LOGIN_WITH_GOOGLE_BTN).click()
+
+    def enter_usernsme_pass(self, username, password):
+        self.driver.find_element_by_id(USER_NAME_IDENTIFIER).send_keys(username + Keys.RETURN)
+        try:
+            self.driver.find_element_by_name(PASSWORD_IDENTIFIER).send_keys(password + Keys.RETURN)
+        except NoSuchElementException:
+            self.logger.error('Under enter_username_pass : User ID is wrong')
```

Comment:
how do you know username is wrong, could be that internet is down or the password_identifier changed in frontend